logicmonitor_data_sdk/api/logs.py

```diff
@@ -165,9 +167,18 @@ def _do_request(self):
       self.UnLock()
 
   def _merge_request(self, single_request):
+    # size limiting
+    payload_cache = json.dumps(self._payload_cache)
```

Comment:
Add a try-catch whenever using json.dumps, as data serialisation is prone to error.